File: Python/mario_less.py
import cs50

while True:
    h = cs50.get_int("Height: ")
    if h > 0 and h <= 8:
        break
for i in range(h):
    for j in range(h):
        if i + j < h - 1:
            print(" ",end='')
        else:
            print("#",end='')
    print()

File: C/mario_less.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{   //check user input
    do
    {
       int h = get_int ("Height: ");
    }
    while (h<1 || h>8);
    //prints the # according to the size specified by the user 'h'
    for (int n=0; n<h; n++)
    {
        for (int j=0; j<h; j++)
        {
            if (n + j < h - 1)
            {
                printf(" ");
            }
            else
            {
                printf("#");
            }

        }
        printf("\n");
    }

}
